### PR TITLE
Add webp support

### PR DESCRIPTION
**Pull request by cemonel**

### CODE REVIEW

File: prism/app.py [hunk 1]
```diff
@@ -152,19 +152,18 @@ def info(path, args, customer):
     return json_response(info)
 
 
-def fetch_image(args, original_url, exists):
-    if args['with_info'] or args['force'] or not exists:
```
Comment (cemonel) [inline prism/app.py:156]: This if check is redundant.
See line 181.

**Review by yigitguler — APPROVED** [review]

**Review by yigitguler — APPROVED** [review]

**Comment by yigitguler** [comment]
Any thoughts @fergalwalsh?

**Review by fergalwalsh — COMMENTED** [review]
Yeah if imagemagick is handling webp fine then this should work.

**Comment by cemonel** [comment]
Yes, Imagemagick handles the rest. 
Before merging this I will add some tests for webp and also some minor documentation changes. Then we can merge this.

**Review by yigitguler — APPROVED** [review]

**Pull request merged**